src/views/HomeView.jsx

```diff
@@ -1,19 +1,38 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import { Link } from "react-router-dom";
 
-const HomeView = ({ match }) => {
+const HomeView = ({ match, uname }) => {
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    fetch(`https://m07.herokuapp.com/`)
+      .then(response => response.json())
+      .then(users => setUsers(users))
+      .catch(err => console.error(err));
+  }, []);
+
+  console.log("hey", users);
+
```

Comment:
I wonder why from line 8 to line 15, you have moved from HomeView to src/use-users-list, but it is still showing here?